incubator/mysql/values.yaml

```diff
@@ -0,0 +1,25 @@
+## Bitnami mysql image version
+## ref: https://hub.docker.com/r/bitnami/mysql/tags/
+##
+imageTag: "5.7"
```

Comment:
The MySQL maintainers re-push this tag if there is an update to `5.7.x`, maybe the chart should be locked to `5.7.14` to ensure reproducibility?